tests/test_commands.py

```diff
@@ -212,17 +212,19 @@ def __init__(self):
             self.send_message = MagicMock()
 
     fake_bot = FakeBot()
-    birthday_boy_or_girl_name = "Tutu Lulul"
-    fake_message.text = '/hbd {}'.format(birthday_boy_or_girl_name)
-    hbd_greetings = "hoi " + birthday_boy_or_girl_name \
-        + " met ultah ya moga sehat dan sukses selalu \xF0\x9F\x8E\x89 \xF0\x9F\x8E\x8A"
+    fake_name = "Tutu Lulul"
+    fake_message.text = '/hbd {}'.format(fake_name)
+    greetings = 'hoi ' \
+                ' met ultah ya moga sehat dan sukses selalu ' \
+                '\xF0\x9F\x8E\x89 \xF0\x9F\x8E\x8A' \
```

Comment:
Did you forget to put `{}` in `greetings`? Anw, breaking long string like this is un-Pythonic. Use this:

```python
greeting_format = ('hoi {} met ultah ya moga sehat dan sukses selalu '
                   '\xF0\x9F\x8E\x89 \xF0\x9F\x8E\x8A')
greeting = greeting_format.format(fake_name)
```